pkg/velaql/view.go

```diff
@@ -128,20 +127,9 @@ func ValidateView(ctx context.Context, viewStr string) error {
 
 	// Make sure `status` or `export` field exists
 	vStatus := val.LookupPath(cue.ParsePath(DefaultExportValue))
-	errStatus := vStatus.Err()
 	vExport := val.LookupPath(cue.ParsePath(KeyWordExport))
-	errExport := vExport.Err()
-	if errStatus != nil && errExport != nil {
-		return errors.Errorf("no `status` or `export` field found in view: %v, %v", errStatus, errExport)
-	}
-	if errStatus == nil {
-		_, errStatus = sets.ToString(vStatus)
-	}
-	if errExport == nil {
-		_, errExport = sets.ToString(vExport)
-	}
-	if errStatus != nil && errExport != nil {
-		return errors.Errorf("connot get string from` status` or `export`: %v, %v", errStatus, errExport)
+	if !vStatus.Exists() && !vExport.Exists() {
+		return errors.Errorf("no `status` or `export` field found in view")
```

Comment:
Shouldnt we give the name of the view so that its easier to debug the correct cue file?

Reply:
This function only verifies CUE, so it will not contain the file name. But the function use this one will report the file name:
```
		obj, err := renderCUEView(ctx, view)
		if err != nil {
			return nil, errors.Wrapf(err, "render velaQL view file %s", view.Name)
		}
```

Reply:
cool,  that helps!